packages/indexer-common/src/indexer-management/resolvers/cost-models.ts

```diff
@@ -68,6 +69,15 @@ export default {
   ): Promise<object> => {
     const update = parseGraphQLCostModel(costModel)
 
+    // Validate cost model
+    try {
+      const modelForValidation = update.model || 'default => 1;'
```

Comment:
This might break in the future. For example this cost model:

`default when $UNDER_LOAD => 1;`

Right now what will happen is that this statement will be compiled to...
`default => error;`

But it may be desirable to propagate a warning or error or something instead.

What should happen here is to load the missing data from the database during a partial update to verify the change is valid.

This will work today though, so carry on...

Reply:
Yeah, for now, this is just a little trick to only focus on the variables if no new model was provided. Of course, if at any point the model already in the database becomes invalid due to a breaking change in agora, that wouldn't be caught here.

Reply:
No, I don't think you understand the point I'm trying to make.

What I'm saying is that the cost model and it's variables may not be able to be validated independently. Whether or not a cost model compiles may depend on the variables. So, by using `{}` here when the model has a partial update may cause an error that would not have been encountered had the variables in the database been used instead.